python/pyspark/ml/recommendation.py

```diff
@@ -26,8 +26,8 @@
 
 
 @inherit_doc
-class ALS(JavaEstimator, HasCheckpointInterval, HasMaxIter, HasPredictionCol, HasRegParam, HasSeed,
-          JavaMLWritable, JavaMLReadable):
+class ALS(JavaEstimator, HasCheckpointInterval, HasMaxIter, HasPredictionCol,
+          HasRegParam, HasSeed, JavaMLWritable, JavaMLReadable):
```

Comment:
If this is just a cleanup change, you might want to revert.  It can be fixed another time and there's already a lot of changes here.